presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -5,6 +5,21 @@ Tuning Presto
 The default Presto settings should work well for most workloads. The following
 information may help you if your cluster is facing a specific performance problem.
 
+When setting up cluster on particular machines for a particular workload you should
+pay particular attention to properly choosing values of the following properties:
+
+  * :ref:`distributed-joins-enabled <tuning-pref-general>`
+  * :ref:`query.max-memory <tuning-pref-query>`
+  * :ref:`query.max-memory-per-node <tuning-pref-query>`
+  * :ref:`query.initial-hash-partitions <tuning-pref-query>`
+  * :ref:`task.concurrency <tuning-pref-task>`
+
+For example, on a 11-node cluster dedicated to Presto (1 Coordinator + 10 Workers) with 8-core CPU and 128GB of RAM per node and low concurrency workload, you might want to set those properties to:
```

Comment:
How do you define low concurrency? And how much is the heap size in jvm.config for this setup?